docs/known_issues.md

```diff
@@ -14,10 +14,23 @@ As a workaround, use `tag.turbo_frame` instead of `turbo_frame_tag`.
 
 Note: For the same functionality as `turbo_frame_tag(my_model)`, use `tag.turbo_frame(id: dom_id(my_model))`.
 
-## form_for compatibility
+## `form_for` compatibility
 
 ViewComponent [isn't compatible](https://github.com/viewcomponent/view_component/issues/241) with `form_for` helpers by default.
 
+This means that it isn't supported to pass a form object to a view component.
+It won't render as expected. Some content will be rendered twice.
+
+Consider following options:
+
+- Render a form end-to-end within one view component
+- Render a [classical view partial](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) within the view component, which includes the form
+- Use [FormBuilder](https://guides.rubyonrails.org/form_helpers.html#customizing-form-builders) to create reusable form components.
```

Comment:
I'm not sure a form builder will help across components. The builder is actually a big part of the reason form helpers aren't compatible with view components, namely because they retain a reference to the view context in which they were called. I was trying to address that [here](https://github.com/ViewComponent/view_component/pull/1642). I suppose you could write your own form helper that does something similar, but there would be no way to know which view context is the "current" one without some additional help from the VC library itself (i.e. what I was trying to do with `@render_stack`).

Or maybe you mean a form builder in the generic sense, i.e. not Rails form builders?

Reply:
I also mean the rails FormBuilder. I know this is an issue.

When I did my research on this topic, I saw multiple repos, where people tried to use view components to style HTML form elements. E.g. to style a textarea field. However, in this case, FormBuilder can be considered a view component implementation of some sort. And maybe this is enough for them and their use case at this moment.

Reply:
Ahh ok I understand. That's an interesting point.